less/pageNav.less

```diff
@@ -30,6 +30,14 @@ html {
     margin: @item-margin;
   }
 
+  &__btn-icon {
+    display: inline-flex;
```

Comment:
I think the issue is resolved with line 34 alone. I can't see that `.icon { display: inline-flex; } `is needed @swashbuck ?

Reply:
Hellooooooo?

Reply:
Sorry, my comments were hidden for some reason. I think they are there now..